### DISCUSSION TITLE
Possible to exclude `#` = H1 level entries from Md ToC?

### DISCUSSION THREAD

**Discussion by dajare** [Zettlr/Zettlr#2877] [Q&A]
Is it possible to exclude `#` = `H1` level entries from the auto-generated Markdown ToC in Zettlr's sidebar?

I typically structure my Markdown documents like this:

```
# Doc Title
## Section 1
### A thing
### Another thing
## Section B
...etc.
```
So, "logically" Section 1 should be the first level of a ToC, like:

```
Doc Title /omit?/
1 Section 1
1.1 A thing
1.2 Another thing
2 Section B
...etc.
```

But what is produced is (of course):

```
1 Doc Title
1.1 Section 1
1.1.1 A thing
1.1.2 Another thing
1.2 Section B
...etc.
```

Like this:

![Generated ToC](https://user-images.githubusercontent.com/40625/144213997-7479458a-80d2-454f-86d3-8c7f8c4cd9d1.png)

So: back to my question — **is it possible to exclude the H1 element from this ToC?** Not crucial, but would be a help (to this user!) if that was configurable. I know that some e.g. export options (to EPUB, for example) include this kind of thing (via Pandoc), so maybe possible here, too?

If not, I suppose there is a work-around in using `H1` where I would normally use `H2`, etc., but ... that runs against the grain! Or perhaps there's a better work-around I haven't thought of? Thanks for any illumination on this.

**Comment by nathanlesage** [comment]
Right now that's not possible, but in general since Pandoc also allows you to shift headings, that idea is plausible I think.

**Comment by ghost** [comment]
I would also like to exclude parts or even the whole ToC when I export things; ToC = false

**Reply by nathanlesage** [reply]
When you export things, the internal ToC in the sidebar doesn't have anything to do. You can add or prevent a Table of Contents within the defaults files in the assets manager.

**Comment by ghost** [comment]
**Edit:** I installed latex and using that PDF export method, it works there flawlessly with ToC: false

 Then I tried the normal PDF export like I have done a thousand times the last days, and it worked there as well all of a sudden. I am done for today.

---

If you mean the default export settings for a PDF file, then it does not work for me with the version 2.1.3. 
I stumbled over this topic, because I am trying to get rid of the ToC in my documents.

https://i.imgur.com/G72hFMb.png

I also have read the manual for pandoc in case I am missing something, but apparently I must do something wrong, and do not know what.

https://pandoc.org/MANUAL.html

**Reply by nathanlesage** [reply]
Uhm, yeah the main problem with PDF Document is that this is actually an HTML export under the hood … so that's confusing because the exporter doesn't use the "PDF" asset for that, but rather the "HTML" asset. I'm aware of that problem!

**Reply by ghost** [reply]
That explains it. Anyway, thank you very much for your support and time.
